bika/lims/browser/js/coffee/bika.lims.bikalisting.coffee

```diff
@@ -239,6 +239,8 @@ window.BikaListingTableView = ->
 
         $(checked).each (e) ->
             transitions = $.parseJSON($(this).attr('data-valid_transitions'))
+            if ! transitions.length
```

Comment:
you could also write `return unless transitions.length`

Reply:
My first coffee code line ever :) Thank you!